src/templates/governance-page/DaoGovernanceDetails/DaoVotes.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC, useEffect, useState } from "react"
+import { gql, request } from "graphql-request"
+import BN from "bignumber.js"
+import dayjs from "dayjs"
+import { PageSection, ResponsiveStack } from "../../../components/PageSection"
+import { H4 } from "../../../components"
+import VoteResultsCard from "./VoteResultsCard"
+import { Proposal, SNAPSHOT_URL, SnapshotProposal } from "./types"
+
+const DaoVotes: FC = () => {
+  const [proposals, setProposals] = useState<Proposal[]>([])
+
+  const getProposals = async () => {
+    const query = gql`
+      {
+        proposals(
+          first: 2
+          skip: 0
+          where: { space_in: ["threshold.eth"] }
+          orderBy: "created"
+          orderDirection: desc
+        ) {
+          id
+          author
+          title
+          link
+          start
+          end
+          state
+          choices
+          scores
+        }
+      }
+    `
+    const result: { proposals: SnapshotProposal[] } = await request(
+      SNAPSHOT_URL,
+      query
+    )
+
+    const scrubbedProposals = result.proposals.map((proposal) => {
+      const dateRange = `${dayjs(proposal.start * 1000).format(
+        "MMM D YYYY"
+      )} - ${dayjs(proposal.end * 1000).format("MMM D YYYY")}`
```

Comment:
I left comment about `dayjs` lib above. I would consider using the `Date` object.